.github/workflows/continuous-integration-workflow.yml

```diff
@@ -1,48 +1,48 @@
-name: Continuous Integration
-on:
-  pull_request:
-    types: [opened, synchronize]
-  push:
-    paths-ignore:
-      - 'README.md'
-
-jobs:
-  build-and-test:
-    name: Build and test
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-    - name: build docker container 
-      run: docker build --target test --tag todo-app:test .
-    - name: run docker tests
-      run: docker run todo-app:test
-      
-  build-and-push-docker-image:
-    name: Docker Build, Tag & Publish
-    needs: build-and-test
-    if: ${{ github.ref == 'refs/heads/main' }}
-    runs-on: ubuntu-latest
-    steps:
-      - name: Git Checkout Code
-        uses: actions/checkout@v1
-        id: git_checkout
-
-      - name: Build, Tag & Publish Image to DockerHub
-        uses: dinushchathurya/build-tag-push-action@v1.2.1
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-          context: .
-          file: Dockerfile
-          repoOwner: ${{ secrets.DOCKERHUB_USERNAME }}
-          repository:  todo-app
-          tag: prod
-
-  refresh-azure-image:
-    name: Refresh Azure Image
-    needs: build-and-push-docker-image
-    runs-on: ubuntu-latest
-    steps:
-      - name: Curl Azure webhook
-        run: |
+name: Continuous Integration

```

Comment:
I'm guessing a bit line-ending change has snuck in somewhere so these all look much more changed than they are - no major issue but one to try avoid when you can (the worst being when two colleagues keep flipping the endings back and forth!)